docs/release-notes.md

```diff
@@ -33,15 +33,26 @@
 
 # Release notes
 
+## Trino Gateway 7 (19 March 2024)
```

Comment:
```suggestion
## Trino Gateway 7 (19 Mar 2024)
```

Reply:
Trino (and also Trino Gateway convention) is to use 3 letters for month - https://trino.io/docs/current/release.html

Reply:
oh .. sorry @ebyhr ... I must have looked at it cross eyed or something .. I thought it was three letters already ..  it is now ;-)